.editorconfig

```diff
@@ -16,6 +16,6 @@ insert_final_newline = true
 [*.md]
 trim_trailing_whitespace = false
 
-[*.{json,yml}]
+[*.{json,md,yml}]
 indent_style = space
```

Comment:
Wait, why are we indenting these with spaces? Sidenote, I know.

However, adding md into this means that markdown and html are inconsistent which annoys me.

Reply:
> Wait, why are we indenting these with spaces? Sidenote, I know.

I don't know, JSON/YAML files always have here. I'm not opposed to changing that (already established that I don't care about tabs vs. spaces as long as some tool helps me not caring about that), but we probably don't want to do it here.

> However, adding md into this means that markdown and html are inconsistent which annoys me.

I agree with you, but I'm not adding `.md` here to change anything, rather to reflect reality. All our Markdown files have spaces, and not doing this change causes lists in Markdown to be indented with 4 spaces, as such:

```md
-   This is the first item of the list
    Second line of the first item
-   This is the second item of the list
```

Yep, `-<3 spaces>Text`... 😞

Reply:
> Wait, why are we indenting these with spaces? Sidenote, I know.

I just remembered one of the key reasons is that npm rewrites `package.json` on dependency install/update, and they do so with 2 spaces. However, it seems like `yarn` respects tabs correctly as opposed to `npm`. Good to know.

Also I just learned that tabs are forbidden in YAML......

So we could theoretically use tabs for JSON, but not for YAML (e.g. Prettier still uses spaces even without this override). However, we'd have to remove `indent_size = 4` from the file for Markdown, but as far as I know tabs are not a thing in Markdown (e.g. Prettier didn't rewrite everything with tabs).

I'm personally happy to remove as many overrides as we can, but it would make the initial diff (and git blame) a little annoying, which is not such a big deal on JSON files IMO. Do you want me to add a commit without this `space` override to see how that looks to you? 🙂